### PR TITLE
Removing nested trace output from trace.go

### PR DESCRIPTION
**Pull request by kris-nova**
My first attempt at a PR..

Closes https://github.com/sdboyer/gps/issues/129

@sdboyer when you get a chance let me know what you think!

### CODE REVIEW

**Comment by kris-nova** [comment]
### Sample output with Kubernetes kops and example.go

```bash
glide install && go build -o gps example.go && chmod +x gps && mv gps ~/Workspace/kops && cd ~/Workspace/kops/ && ./gps && rm -rf .repocache; cd -
```

[Output here](https://gist.github.com/kris-nova/3b5c19ddba92870405a1417112cf8476)

**Comment by sdboyer** [comment]
Huh. Well. Simplest really is best - I think I might like that more than any of the other options I discussed in #129 :)

One thing, though - I think it might still be valuable to indent the "inner" rows - things not starting with "select", "attempt", "revisit" or "backtrack". Yes, you can infer it from the counter on the left, but that's not the most useful for visual scanning and grouping. I'd at least like to see if doing it that way provides an improved experience.

Also, that leading pipe feels unnecessary now - maybe we can drop that, too.

File: trace.go [hunk 1]
```diff
@@ -165,10 +165,22 @@ func (s *solver) traceInfo(args ...interface{}) {
...
 
-	prefix := strings.Repeat("| ", preflen)
+	prefix := getprei(preflen)
 	s.tl.Printf("%s\n", tracePrefix(msg, prefix, prefix))
 }
 
+func getprei(i int) string {
+	var s string
+	if i < 10 {
+		s = fmt.Sprintf("(%d)   ", i)
```
Comment (sdboyer) [inline trace.go]: At this point, might it be safer/better to just use a tab?

**Comment by kris-nova** [comment]
@sdboyer Pushed up changes per your review.. Totally agree about the pipe

I think I got all of the "inner" messages - but if you could check that would be helpful

Also I indented the "inner" messages over 2 spaces.. I tried a few options, and that looked the best. Sample output:

```
Root project is "/Users/kris/Workspace/kops"
 69 transitively valid internal packages
 132 external packages imported from 17 projects
(0)   ✓ select (root)
(1)   ? attempt github.com/ghodss/yaml with 1 pkgs; 1 versions to try
(1)       try github.com/ghodss/yaml@master
(1)   ✓ select github.com/ghodss/yaml@master w/1 pkgs
(2)   ? attempt github.com/sergi/go-diff with 1 pkgs; 1 versions to try
(2)       try github.com/sergi/go-diff@master
(2)   ✓ select github.com/sergi/go-diff@master w/1 pkgs
(3)   ? attempt github.com/golang/glog with 1 pkgs; 1 versions to try
(3)       try github.com/golang/glog@master
(3)   ✓ select github.com/golang/glog@master w/1 pkgs
(4)   ? attempt github.com/spf13/viper with 1 pkgs; 1 versions to try
(4)       try github.com/spf13/viper@master
(4)   ✓ select github.com/spf13/viper@master w/1 pkgs
(5)   ? attempt github.com/mitchellh/mapstructure with 1 pkgs; 1 versions to try
(5)       try github.com/mitchellh/mapstructure@master
(5)   ✓ select github.com/mitchellh/mapstructure@master w/1 pkgs
(6)   ? attempt github.com/spf13/afero with 1 pkgs; 1 versions to try
(6)       try github.com/spf13/afero@master
(6)   ✓ select github.com/spf13/afero@master w/1 pkgs
(7)   ? attempt github.com/spf13/cobra with 2 pkgs; 1 versions to try
(7)       try github.com/spf13/cobra@master
(7)   ✓ select github.com/spf13/cobra@master w/2 pkgs
(8)   ? attempt github.com/inconshreveable/mousetrap with 1 pkgs; 1 versions to try
(8)       try github.com/inconshreveable/mousetrap@master
(8)   ✓ select github.com/inconshreveable/mousetrap@master w/1 pkgs
(9)   ? attempt golang.org/x/text with 2 pkgs; 1 versions to try
(9)       try golang.org/x/text@master

```

File: trace.go [hunk 2]
```diff
@@ -152,23 +156,35 @@ func (s *solver) traceInfo(args ...interface{}) {
...
-		msg = tracePrefix(data.traceString(), "| ", failCharSp)
+		msg = tracePrefix(innerIndent + data.traceString(), "  ", failCharSp)
 	case error:
 		// Regular error; still use the x leader but default Error() string
-		msg = tracePrefix(data.Error(), "| ", failCharSp)
+		msg = tracePrefix(innerIndent + data.Error(), "  ", failCharSp)
 	default:
 		// panic here because this can *only* mean a stupid internal bug
-		panic(fmt.Sprintf("canary - unknown type passed as first param to traceInfo %T", data))
+		panic(fmt.Sprintf("%scanary - unknown type passed as first param to traceInfo %T", innerIndent, data))
```
Comment (sdboyer) [inline trace.go]: nit: no need to indent on the panic

Comment (kris-nova) [inline trace.go]: Ah good catch - fixed 😄

Comment (sdboyer) [inline trace.go]: still have a `%` :P

Comment (kris-nova) [inline trace.go]: SMH - Should I squash?

**Comment by sdboyer** [comment]
Yep, we have a winner! 🎉

**Pull request merged**